### ISSUE TITLE
High Speed Blower in HA?

### ISSUE THREAD

**Issue by jurrianv** [garbled1/balboa_homeassistan#6]
Hi,

I have 2 pumps in my SPa. Both pumps are OFF - LOW - HIGH. I can only operate OFF or LOW with the switch. How can I get the pump into HIGH speed?

**Comment by natekspencer** [comment]
@jurrianv, if you open the pump entity, you should be able to choose a speed in addition to just turning on/off the pump.

**Comment by scottyphillips** [comment]
@natekspencer thanks for the tip, i'm thinking that longer term 2 speed pumps could be a candidate for the new 'select' entity recently added to Home Assistant.

**Comment by natekspencer** [comment]
@scottyphillips agreed, I saw that announcement and thought of this integration specifically, but haven't jumped in to code it yet.

**Comment by scottyphillips** [comment]
I coded it recently for my HVAC module, works well and its not too complex.

**Comment by garbled1** [comment]
Oh.. I would be very interested in a select type like that.. thats a good idea...
